pages/common/sftp.md

```diff
@@ -11,6 +11,10 @@
 
 `sftp -P {{remote_port}} {{remote_host}}`
 
+- Navigate through the LOCAL system file hierarchy:
+
+`lpwd, lls, lcd`
```

Comment:
I think these commands are pretty easy to catch up with. User who intended to use `sftp` will guess that `lpwd` stands for _local present working directory_.  I'm just trying to keep thing concise. Please let me know if I have to split them.